app/root/portalTheme.ts

```diff
@@ -193,33 +169,9 @@ export const portalTheme: MantineThemeOverride = {
       },
     },
     Card: {
-      styles: (theme) => ({
+      styles: () => ({
         root: {
-          padding: "32px",
-          backgroundColor:
```

Comment:
im not clear why we dont need all this now

Reply:
See here https://github.com/pokt-foundation/portal-platform/blob/1554c2edb1167d3f17a23ef3c1b0d6ba0d8b0182/app/styles/root.css#L7

Instead of setting the `backgroundColor`, I'm setting the variable `--mantine-color-body` to the value we want... this is way easier and instead of setting the background color for all the components (Drawer, Modal, card..etc) It's automatically handled this way.